diff --git a/dist/vite.js b/dist/vite.js
index a33eb5b7c32534b106b3d0c615914cae9b1d8c42..c3d925b0b59eea3c8c8cc0d125ff4ba829e99244 100644
--- a/dist/vite.js
+++ b/dist/vite.js
@@ -196,6 +196,7 @@ async function bundle(environment, bundledAssets, outfile) {
 	} else {
 		buildOptions.platform = "node";
 		buildOptions.outfile = destination.replace(/\.js$/, ".mjs");
+		buildOptions.external = ["lightningcss", "fsevents"];
 		buildOptions.banner = { js: `import { createRequire as topLevelCreateRequire } from 'module';
 import { dirname as topLevelDirname } from 'path';
 import { fileURLToPath as topLevelFileURLToPath } from 'url';
